extensions/github/src/search-repositories.tsx

```diff
@@ -45,6 +45,14 @@ function SearchRepositories() {
     { keepPreviousData: true },
   );
 
+  useEffect(() => {
```

Comment:
We could use a comment above the code to explain why this piece of code is useful. It may not be obvious at first.

Reply:
Added.